### PR TITLE
Revise root Dockerfile

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Cleanup & updating to match some other root Dockerfiles

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,41 +1,21 @@
...
 
-RUN useradd console-operator
-USER console-operator
+# finally to enable install integration
+#LABEL io.openshift.release.operator true
 
-ENTRYPOINT []
-CMD ["/usr/bin/console-operator"]
\ No newline at end of file
+# entrypoint specified in 03-operator.yaml as `console-operator`
```
Comment (spadgett) [inline Dockerfile]: I'd probably leave `CMD` here

Comment (benjaminapetersen) [reply Dockerfile]: Works for me, updated.

File: Dockerfile [hunk 2]
```diff
@@ -1,41 +1,22 @@
...
+CMD console-operator
```
Comment (spadgett) [inline Dockerfile]: Better to make it an array with the full path as before

Comment (benjaminapetersen) [reply Dockerfile]: Its now building in the root dir I believe, but I can still update to array.

Comment (benjaminapetersen) [reply Dockerfile]: ah no i did put it back in `/usr/bin`.

Comment (benjaminapetersen) [reply Dockerfile]: testing this now

Comment (benjaminapetersen) [reply Dockerfile]: Quotes fixed.  Heh.

**Pull request merged**